### PR TITLE
Adding Advanced configuration support for service

### PR DESCRIPTION
**Pull request by nareshkakubal**
* This commit addresses advanced configuration for Web Service.
* Fixes web service resource import issues.
* Adding examples and updating documentation.

### CODE REVIEW

File: banyan/resource_service_db.go [hunk 1]
```diff
@@ -71,6 +71,12 @@ func DbSchema() map[string]*schema.Schema {
 			Required:    true,
 			Description: "The external-facing network address for this service; ex. website.example.com",
 		},
+		"suppress_device_trust_verification": {
+			Type:        schema.TypeBool,
+			Description: "If device trust verification is suppressed, mobile devices will not be able to pass the device trust check.",
```
Comment (yoshiotu) [inline banyan/resource_service_db.go:76]: Did the text for these Descriptions come from our documentation site? I am not sure they make sense.

File: banyan/resource_service_web.go [hunk 2]
```diff
@@ -254,12 +423,37 @@ func expandWebBackend(d *schema.ResourceData) (backend service.Backend) {
...
+		items[i] = raw.(string)
+	}
+	return items
+}
+
+func expandBackendDNSOverrides(d *schema.ResourceData) map[string]string {
+	dnsOverrides := make(map[string]string)
+	v, ok := d.GetOk("dns_overrides")
+
+	if !ok || len(v.(map[string]interface{})) <= 0 {
```
Comment (yoshiotu) [inline banyan/resource_service_web.go:446]: Why `<= 0` instead of `== 0`? `len()` is always non-negative.

Comment (nareshkakubal) [reply banyan/resource_service_web.go:446]: sure @yoshiotu i should change to ==

File: banyan/resource_service_db.go (hunk 1 above)
Comment (nareshkakubal) [reply banyan/resource_service_db.go:76]: @yoshiotu no i would need proper text

**Review by yoshiotu — APPROVED** [review]

**Pull request merged**